src/encoded/schemas/treatment.json

```diff
@@ -1,5 +1,6 @@
 {
     "title": "Treatment",
+    "description": "A non-standard treatment applied during analyte preparation or library preparation.",
```

Comment:
What does "non-standard" mean in this context? Is the implication that "standard" treatments would fall under PreparationKit?

Reply:
That was the intention, although I am still having a hard time coming up with examples of a Treatment. Maybe "A treatment not from a kit"?